targets/csharp/source/PlayFabSDK/source/Uunit/PluginManagerTest.cs

```diff
@@ -54,9 +54,11 @@ public void PluginManagerDefaultPlugins(UUnitTestContext testContext)
         {
             var playFabSerializer = PluginManager.GetPlugin<ISerializerPlugin>(PluginContract.PlayFab_Serializer);
             var playFabTransport = PluginManager.GetPlugin<ITransportPlugin>(PluginContract.PlayFab_Transport);
+            var playFabTransportWithPolly = PluginManager.GetPlugin<ITransportPlugin>(PluginContract.PlayFab_TransportWithPolly);
```

Comment:
Again, I don't think Polly should be a separate type of default plugin. It could eventually be the default Transport plugin, but when that happens it would be using the PlayFab_Transport contract.